custom-vu/src/main/kotlin/jces1209/vu/action/WorkAnIssue.kt

```diff
@@ -19,7 +19,10 @@ class WorkAnIssue(
     private val random: SeededRandom,
     private val editProbability: Float,
     private val commentProbability: Float,
-    private val linkIssueProbability: Float
+    private val linkIssueProbability: Float,
+    private val changeAssigneeProbability: Float,
+    private val transitionProbability: Float,
+    private val mentionUserProbability: Float
```

Comment:
It seems the PR is about changing assignees, so the unrelated transitions and mentioning code should be left out from the diff.